### ISSUE TITLE
Problem with Get-TssFolderPermissionStub tests

### ISSUE THREAD

**Issue by peetrike** [thycotic-ps/thycotic.secretserver#79]
**Describe the bug**
Running tests end with Invalid URI exception

**To Reproduce**
1. PowerShell script:

```powershell
 Invoke-Pester -Path .\tests\folder-permissions\

```

1. Error/issue observed:

```powershell
Starting discovery in 1 files.
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.CheckProtocol(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.PrepareUri(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.GetRequest(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.ProcessRecord()
   at System.Management.Automation.CommandProcessor.ProcessRecord()
at Invoke-RestApi<Process>, C:\Projects\GitHub\thycotic.secretserver\src\functions\Invoke-RestApi.ps1: line 131
at <ScriptBlock>, C:\Projects\GitHub\thycotic.secretserver\tests\folder-permissions\Get-TssFolderPermissionsStub.Tests.ps1: line 34
```

**Cause**
It seems, that problem is in:
https://github.com/thycotic-ps/thycotic.secretserver/blob/23b93afacf47312c8c3d2f20104a617f790f4bc7/tests/folder-permissions/Get-TssFolderPermissionsStub.Tests.ps1#L30

Replacing `$ssv3` with `$ss` resolves the problem

```powershell
Name                           Value
----                           -----
PSVersion                      7.1.2
PSEdition                      Core
GitCommitId                    7.1.2
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

***Host information:***

[ ] Powershell.exe
[x] pwsh.exe
[ ] PowerShell ISE
[ ] VS Code (Insiders/Stable)

**Additional context**

**Comment by wsmelton** [comment]
This is resolved and will be pushed up to the repository by this afternoon.

**Comment by wsmelton** [comment]
Fixed in  7eeb8d8 

https://github.com/thycotic-ps/thycotic.secretserver/blob/7eeb8d87783833502c1e47650f66e611df71aae1/tests/folder-permissions/Get-TssFolderPermissionsStub.Tests.ps1#L36
